andela/AndelaEatsUI
# #161216668 Split admin orders page

#### What does this PR do?
- Splits Admin orders page into order history and `export page` for orders in the last 24 hours.

#### Description of Task to be completed?
- Split admin orders page for export.
- Add pagination to pages.

#### How should this be manually tested?
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-view-last-24hrs-orders-161216668`
- Run these commands `npm run bundle` and `npm run dev` to setup app
- Log in as an `Admin` and ensure that you see the order history page
- Click on export button to ensure you are redirected to the orders in last 24hours page
- Ensure pagination is on both pages
- Ensure all tests are passing by running `npm run test-dev`

#### What are the relevant pivotal tracker stories?
- [#161216668](https://www.pivotaltracker.com/story/show/161216668)

#### Screenshots (If Applicable)
<img width="1440" alt="screen shot 2018-10-17 at 12 51 54 am" src="https://user-images.githubusercontent.com/32123313/47053863-ddc12880-d1a6-11e8-90a0-3cb620e33763.png">
<img width="1440" alt="screen shot 2018-10-18 at 10 26 44 am" src="https://user-images.githubusercontent.com/32123313/47144882-5ff14000-d2c0-11e8-8b12-9ebc0939b3ea.png">
